dbs/mongo/queries/markQuestion.js

```diff
@@ -0,0 +1,27 @@
+const {Questions} = require('../models.js');
+
+async function markQuestion(q_id, status){
+  if (!q_id && typeof q_id !== 'number') {
+    throw 'Invalide question id'
+  }
+
+  if (status === 'helpful') {
+   await Questions.findOneAndUpdate({question_id: q_id}, {$inc : {question_helpfulness : 1}})
+   .then(res => {
```

Comment:
await and then block? Is findOneAndUpdate a promise? If so, do you need await?

Reply:
You are right, not needed, just got async/await happy